### ISSUE TITLE
How to Run files in Remix Desktop

### ISSUE THREAD

**Issue by pstephenwille** [RareSkills/Solidity-Exercises#19]
Total noob to this tech stack.
I can not get these files to load and run in Remix desktop.  My expectation is that I can import the dir into a Workspace and then run the test files separately either in the GUI or CLI.  However, Remix does not seem to work this way and will only allow me to import individual files.  Since the RareSkills Solidity tutorial recommends Remix, I would suggest adding instructions on how to dev these test problems in Remix destkop, or even the web IDE would help.

To recap, with these source code examples, I would expect  instructions on how to run them in Remix; Web/Desktop.

**Comment by RareSkills** [comment]
Remix is only suitable for testing out a single file at a time -- just follow the steps in the tutorial for that.

For everything else, Foundry is much easier.
